pkg/controller/vsphere/actuator_test.go

```diff
@@ -100,9 +101,7 @@ func TestMachineEvents(t *testing.T) {
 		},
 	}
 	g.Expect(k8sClient.Create(context.Background(), configNamespace)).To(Succeed())
-	defer func() {
-		g.Expect(k8sClient.Delete(context.Background(), configNamespace)).To(Succeed())
-	}()
+	// Cannot delete openshift-config so not setting up defer to remove it
```

Comment:
I think the name of the namespace being used was changed, so perhaps we can revert this section

Reply:
Ok, I'll clean up and retry.

Reply:
Looks good, I removed this change now.